src/mumble/pluginAPI/PluginComponents.h

```diff
@@ -0,0 +1,134 @@
+/**
+ * This header file contains definitions of types and other components used in Mumble's plugin system
+ */
+
+#ifndef MUMBLE_PLUGINCOMPONENT_H_
+#define MUMBLE_PLUGINCOMPONENT_H_
+
+#include <stdint.h>
+#include <stddef.h>
+
+#define STATUS_OK 0
+
+enum TalkingState {
```

Comment:
I would prefer a enum class, not sure what Mumble's code style says about that?

Reply:
Okey seems like mumble uses plain enum too, enum class would also be C++ not C.
https://github.com/mumble-voip/mumble/search?l=C%2B%2B&q=enum

Reply:
> enum class would also be C++ not C

That alone would make it impossible to use an enum _class_, wouldn't it?